library/styleguideComponents/ComponentsList/index.js

```diff
@@ -40,11 +42,17 @@ export default class ComponentsList extends Component {
                 <VerticalSection>
                     <VerticalItem
                         name="GettingStarted"
-                        selectedItem={selectedItem}
                         label="Getting Started"
                         href="/#/GettingStarted" />
                 </VerticalSection>
                 <Sections selectedItem={selectedItem} items={items} searchTerm={searchTerm} />
+
+                <VerticalSection>
+                    <VerticalItem
```

Comment:
let's put this inside the VerticalSection above and below the GettingStarted VerticalItem in order to look better while we change the side bar